astropy/utils/tests/test_codegen.py

```diff
@@ -36,6 +36,7 @@ def crashy_function(*args, **kwargs):
         # make_function_with_signature call was one
         tb_lines = traceback.format_tb(tb)
         assert '1 / 0' in tb_lines[-1]
-        assert line in tb_lines[-2] and 'line =' not in tb_lines[-2]
+        # The deprecated decorator breaks this check
+        # assert line in tb_lines[-2] and 'line =' not in tb_lines[-2]
```

Comment:
any chance to fix this?

Reply:
I tried to work out what was causing it and given the function it is testing is now deprecated, I didn't think it was worth the fight.

Reply:
then remove it?

Reply:
Fixed this...